R/sim_gs_n.R

```diff
@@ -301,10 +305,12 @@ sim_gs_n <- function(
 
       # rbind simulation results for all IA(s) and FA in 1 simulation
       ans_1sim <- rbind(ans_1sim, ans_1sim_new)
+
     }
 
-    ans <- rbind(ans, ans_1sim)
+    ans_1sim
   }
+  # ans <- convert_list_to_df_w_list_cols(ans)
```

Comment:
Avoid commented out code to minimize distractions and outdated information.

The real question is, is doing `convert_list_to_df_w_list_cols()` still useful here as in the original implementation? That is, what was the intention.

Reply:
Code has been removed. 
It is now applied over each loop separately rather than to the returned and binded result at the end. The resulting data.frame is now equivalent to what was previously delivered. This is why it was removed at the end.

Reply:
Great!